components/form.rst

```diff
@@ -744,6 +744,11 @@ method to access the list of errors. It returns a
     // a FormErrorIterator instance representing the form tree structure
     $errors = $form->getErrors(true, false);
 
+Clearing Form Errors
+~~~~~~~~~~~~~~~~~~~~
+
+Any errors can be manually cleared using the :method:`Symfony\\Component\\Form\\ClearableErrorsInterface::clearErrors` method. This is useful when you'd like to validate the form without showing validation errors to the user (i.e. during a partial AJAX submission or :doc:`dynamic form modification </form/dynamic_form_modification>`).
```

Comment:
We should precise that clearing the errors makes the form valid. So `clearErrors()` should be called after testing if the form is valid or at least not rely on its state.

Reply:
I have added a new sentence to clarify that.  Thanks!